tasks/publish.yaml

```diff
@@ -1,6 +1,6 @@
 variables:
   - name: TARGET_REPO
-    default: oci://ghcr.io/defenseunicorns/packages/uds
+    default: ghcr.io/defenseunicorns/packages/uds
```

Comment:
if the TARGET_REPO variables were actually getting passed down this would be a breaking change but as it stands today i believe no impact

Reply:
We could bump the version of this repo if you wanted - I did check and it isn't used since that is a very canonical location at this point.

Reply:
More just noting. I think im fine with it since it will be a new tag anyways and technically it would be a future update to uds-cli that would cause the issue